packages/pwa-kit-runtime/package-lock.json

```diff
@@ -19,6 +19,7 @@
         "http-proxy-middleware": "^2.0.6",
         "merge-descriptors": "^1.0.1",
         "morgan": "^1.10.0",
+        "path-to-regexp": "^6.2.1",
```

Comment:
@vcua-mobify let's update the package lock file, since we no longer use this library.